### DISCUSSION TITLE
Discussion channel

### DISCUSSION THREAD

**Discussion by alberk8** [elsa-workflows/elsa-core#4116] [General]
Is there any other discussion channel beside this?  A discord server would be good.  Thank you.

**Comment by sfmskywalker** [comment]
Hello @alberk8 

Thank you for reaching out! Yes, we do have several other communication channels where you can join discussions and get assistance:

- **Discord**: [Join our Discord server](https://discord.com/invite/hhChk5H472)
- **StackOverflow**: You can ask questions tagged with [elsa-workflows on StackOverflow](http://stackoverflow.com/questions/tagged/elsa-workflows).
- **Reddit**: We also have a subreddit at [r/elsaworkflows](https://www.reddit.com/r/elsaworkflows/).

Feel free to join any of these platforms, and we'll be happy to assist you further!
